tox.ini

```diff
@@ -1,4 +1,4 @@
 [flake8]
-
+ignore = E402,E731
```

Comment:
Can you add a comment to this file noting what these errors are so it's clear what isn't being checked?